README.md

```diff
@@ -43,3 +43,17 @@ Go into the package directory e.g. `user` then use `go test -v` within that dire
 
 Specify the user ID for the demo account to automatically share with a new signup with VCA.
 ```
+
+## Running the service locally
```

Comment:
This is not required anymore - it can be deleted.

Reply:
removed